plugins/tutor-contrib-learner-dashboard-mfe/tutor_learner_dashboard_mfe/plugin.py

```diff
@@ -56,6 +56,20 @@ def _add_my_mfe(mfes):
     hooks.Filters.CLI_DO_INIT_TASKS.add_item((service, init_task))
 
 
+########################################
+# DOCKER IMAGE MANAGEMENT
+########################################
+
+# Add the -dev image.
+hooks.Filters.IMAGES_BUILD.add_item(
+    (
+        "learner-dashboard-dev",
+        ("plugins", "mfe", "build", "mfe"),
+        "{{ DOCKER_REGISTRY }}overhangio/openedx-learner-dashboard-dev:{{ MFE_VERSION }}",
```

Comment:
Are we allowed to use overhangio as the namespace for this?

Reply:
For now, it seems [we have to](https://github.com/overhangio/tutor-mfe/blob/master/tutormfe/patches/local-docker-compose-dev-services#L4), unfortunately.